beanstalk_worker/worker_dockerfile_lint.py

```diff
@@ -57,9 +58,11 @@ def lint_job_data(job_data):
     logger.log(level=logging.INFO, msg="Dockerfile Lint check done")
 
     response = {
-        "logs": out,
+        "logs": None,
+        "linter_results": out,
         "action": "notify_user",
```

Comment:
This is going to break the flow, with action `notify_user` and `logs` in the job info, the notify user worker is going to identify the job as failed build and send out the failed build email, with dockerfile linter results as build failure logs in a log file, which is incorrect and job will never get built.

Check the condition about logs and action in this worker https://github.com/CentOS/container-pipeline-service/blob/master/mail_service/worker_notify_user.py#L14